compiler/src/main/kotlin/dev/zacsweers/metro/compiler/ir/IrContributedGraphGenerator.kt

```diff
@@ -238,6 +237,7 @@ internal class IrContributedGraphGenerator(
               .apply {
                 overriddenSymbols += originalGetter.symbol
                 copyAnnotationsFrom(member.ir)
+                extensionReceiverParameter = originalGetter.extensionReceiverParameter
```

Comment:
Do we need this on the property itself too?

Reply:
Good question. At the moment I don't have a repro/failure case indicating it is needed. I'm also not sure if there is an equivalent API available for the property itself. I see that we can set `thisReceiver` although I don't think that's the one we want and it blows up a lot of tests when I tried it right now 😁.

Reply:
Can revisit later 🤷. Wish there was just a `IrOverridableDeclaration.overrideIn(targetContainer)` API

Reply:
Yea I agree, would be nice to be able to duplicate methods/properties in their entirety instead of via builders